doc/develop/microscope.rst

```diff
@@ -89,7 +90,57 @@ Detectors:
    only in confocal microscopes (photo-detector0, photo-detectorN).
 
 System:
- * lens: Contains parameters concerning the parabolic mirror and the lens system.
+ * lens: Defines the optical parameters (e.g magnification) of the optical path.
+
+Roles of components found in the CryoSECOM (ENZEL) system:
+----------------------------------------------------------
+The role of the microscope is cryo-secom.
+
+Emitters:
+ * e-beam: Electron beam of the SEM to scan the sample.
+ * light: Controls the excitation light of the fluorescence microscope.
+
+Actuators:
+ * ebeam-focus: Changes the focus position of the e-beam. It has one axis: z. 
+ * filter: Emission filter to select a specific wavelength band.
+ * stigmator: Controls the rotation of the astigmatic lens. It has one axis rz. It has one metadata:
+
+    1. POS_COR: Position correction to reduce astigmatism.
```

Comment:
Drop. That's used for calibration by the current component class, but not required on a ENZEL in general.